core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -70,7 +70,10 @@
         DISCREPANCY("Discrepancy"),
         COMPLETED("Completed"),
         NULL("null"),
-        MISSING("");
+        MISSING(""),
+        NA("NA"),
+        N_A("N/A"),
+        UNKNOWN("unknown");
 
```

Comment:
would be good to add a unit test for this. See also https://github.com/cBioPortal/cbioportal/pull/1398/commits/392edfc97fafccf957ceb3cb5ac1682e2129c360

You will discover that your change for `N/A` does *not* work and that `N/A` will only work if you also make changes to the `MissingAttributeValues.has` function below.